apps/hash/README.md

```diff
@@ -266,8 +265,9 @@ See `yarn test:playwright --help` for more info.
 
 ### Unit tests
 
-Unit tests are executed by [Jest](https://jestjs.io) and use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/) to cover the UI.
-They can be launched at any time with this command:
+Unit tests are executed by [Vitest](https://vitest.dev/), which we use in place of Jest, due to its improved WebAssembly compatibility.
```

Comment:
it's also primarily because of better TS/ESM support

Reply:
I had actually meant to write ESM support, not WASM 🙃  2am brain... thanks for flagging. Addressed in https://github.com/hashintel/hash/pull/4207/commits/b7f310e0b5b3aba62bf621f5a340bcb4c4944d93